src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/AvaloniaXamlIlLanguageParseIntrinsics.cs

```diff
@@ -221,6 +221,32 @@ public static bool TryConvert(AstTransformationContext context, IXamlAstValueNod
                 }
             }
 
+            if (type.Equals(types.TextTrimming))
```

Comment:
IMO we should add `[XamlEnum]` and `[XamlEnumMember]` attributes and look for those instead to make it more generic. Also easier for IDE's to add support for